third_party/libusb/webport/src/libusb-proxy-receiver.js

```diff
@@ -0,0 +1,87 @@
+/**
+ * @license
+ * Copyright 2021 Google Inc.
+ *
+ * This library is free software; you can redistribute it and/or
+ * modify it under the terms of the GNU Lesser General Public
+ * License as published by the Free Software Foundation; either
+ * version 2.1 of the License, or (at your option) any later version.
+ *
+ * This library is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
+ * Lesser General Public License for more details.
+ *
+ * You should have received a copy of the GNU Lesser General Public
+ * License along with this library; if not, write to the Free Software
+ * Foundation, Inc., 51 Franklin Street, Fifth Floor, Boston, MA 02110-1301 USA
+ */
+
+goog.provide('GoogleSmartCard.LibusbProxyReceiver');
+
+goog.require('GoogleSmartCard.DebugDump');
+goog.require('GoogleSmartCard.Libusb.ChromeUsbBackend');
+goog.require('GoogleSmartCard.Logging');
+goog.require('GoogleSmartCard.RemoteCallMessage');
+goog.require('goog.Promise');
+goog.require('goog.log');
+goog.require('goog.messaging.AbstractChannel');
+
+goog.scope(function() {
+
+const EXECUTABLE_MODULE_REQUESTER_NAME = 'libusb';
```

Comment:
nit: Perhaps add a comment here that this needs to match the string in api_bridge.h